modules/MIPS/measure_activate.php

```diff
@@ -128,7 +128,7 @@
 	font-size: 9pt;
 }
 		</style>
-		<script type="text/javascript" src="../../../library/js/jquery-1.9.1.min.js"></script>
+		<script type="text/javascript" src="../../library/js/jquery-1.9.1.min.js"></script>
```

Comment:
Some of it is done in a module header file.  There is still more of this to be handled.  I have been getting it up to the point where it actually fires.  Guess we are there now!